third_party/zlib/CMakeLists.txt

```diff
@@ -27,8 +27,8 @@ endforeach()
 ExternalProject_add(
     zlib
     GIT_REPOSITORY https://github.com/madler/zlib
-    GIT_TAG v1.2.12
+    GIT_TAG v1.2.13
     PREFIX zlib
-    PATCH_COMMAND git checkout . && git apply ${PROJECT_SOURCE_DIR}/build_shared_libs.patch
+    PATCH_COMMAND git checkout . && git apply ${PROJECT_SOURCE_DIR}/build_shared_libs.patch && git apply ${PROJECT_SOURCE_DIR}/bump_cmake_min_version.patch
```

Comment:
Note (but it does not matter much). You can add as many `COMMAND` as you want between the "key" commands. So this should work:

```cmake
PATCH_COMMAND git checkout . && git apply ${PROJECT_SOURCE_DIR}/build_shared_libs.patch
COMMAND git apply ${PROJECT_SOURCE_DIR}/bump_cmake_min_version.patch
```

See the [Miscellaneous Options](https://cmake.org/cmake/help/latest/module/ExternalProject.html) in the docs:

> Any of the other ..._COMMAND options can have additional commands appended to them by following them with as many COMMAND ... options as needed

Reply:
Yer, on the other hand I like having all of the 3 commands together wrapped as "the" patch command.